packages/backend-resolver/src/server_builder.ts

```diff
@@ -134,8 +134,8 @@ function resolveNavData(nav: NavPath | null): NavResult | null {
             };
         case "bis":
             return {
-                preloadUrl: getBisSheetFetchUrl(nav.job, nav.expac, nav.sheet),
-                sheetData: getBisSheet(nav.job, nav.expac, nav.sheet).then(JSON.parse),
+                preloadUrl: getBisSheetFetchUrl(nav.job, nav.folder ? nav.folder : "", nav.sheet),
+                sheetData: getBisSheet(nav.job, nav.folder ? nav.folder : "", nav.sheet).then(JSON.parse),
```

Comment:
```suggestion
                preloadUrl: getBisSheetFetchUrl(nav.job, nav.folder ?? "", nav.sheet),
                sheetData: getBisSheet(nav.job, nav.folder ?? "", nav.sheet).then(JSON.parse),
```
This seems like exactly the application for the nullish coalescing operator (assuming that `nav.folder` is valid or nullish here --- I wish it were a bit clearer by type).

Reply:
Good catch! Fixed.